src/types/enums.ts

```diff
@@ -0,0 +1,9 @@
+export enum ErrorMessages {
+  LOAD_TODOS = 'Unable to load todos',
+}
```

Comment:
you can write all types of Error here and use it like this `ErrorMessages.LOAD_TODOS`